spec/services/copy_vacancy_spec.rb

```diff
@@ -26,5 +26,28 @@
 
       Timecop.return
     end
+
+    context 'not all fields are copied' do
+      it 'should not copy fields that should be unique' do
+        vacancy = create(:vacancy,
+                         job_title: 'Maths teacher',
+                         slug: 'maths-teacher',
+                         weekly_pageviews: 4,
+                         total_pageviews: 4,
+                         weekly_pageviews_updated_at: Time.zone.today - 5.days,
+                         total_pageviews_updated_at: Time.zone.today - 5.days,
+                         total_get_more_info_clicks: 6,
+                         total_get_more_info_clicks_updated_at: Time.zone.today - 5.days)
+
+        result = described_class.new(vacancy).call
+        expect(Vacancy.find(result.id).slug).to_not eq('maths_teacher')
```

Comment:
The slug above has a dash?

Reply:
Fixed in https://github.com/DFE-Digital/teacher-vacancy-service/pull/942/commits/39703520053b37b6e1a9095598d71df36b0bfd75